.pre-commit-config.yaml

```diff
@@ -42,7 +42,8 @@ repos:
     rev: v1.19.0
     hooks:
       - id: mypy
-        exclude: 'tests/.*'
+        args: [src, --ignore-missing-imports]
+        pass_filenames: false
```

Comment:
Without this, `mypy` doesn't pick up the type stubs which is annoying. This is the issue described in python/mypy#13916 and uses the fix suggested there.